src/convergence.jl

```diff
@@ -95,6 +95,48 @@ function analyticless_test_convergence(dts::AbstractArray,
   ConvergenceSimulation(solutions,dts,auxdata=auxdata,additional_errors=additional_errors)
 end
 
+function analyticless_test_convergence(dts::AbstractArray,
+  prob::AbstractSDDEProblem,
```

Comment:
I don't think this needs a separate dispatch. The other one can get a `Union` on `prob`, and then just check `isa` for what kind of problem to make.